subprojects/gdk_rust/gdk_electrum/src/store.rs

```diff
@@ -87,19 +84,19 @@ pub struct RawStore {
     /// wallet settings
     settings: Option<Settings>,
 
-    /// transaction memos
-    memos: HashMap<bitcoin::Txid, String>,
+    /// account names
+    account_names: HashMap<u32, String>,
```

Comment:
This means that we need to address "Database migration for existing user labels"
do you want to do this in the same MR or in a separate one?